docs/data/base/components/input/input.md

```diff
@@ -14,6 +14,8 @@ githubLabel: 'component: input'
 An input is a UI element that accepts text data from the user.
 The `InputUnstyled` component replaces the native HTML `<input>` tag, and can also be transformed into a `<textarea>` as needed.
 
+{{"demo": "UnstyledInputIntroduction.tsx", "defaultCodeOpen": false, "bg": "gradient"}}
```

Comment:
Did you consider rendering this demo before everything else and a bit smaller in height? I had a quick try at it, here is what the difference looks like:

**Current**:

<img width="1384" alt="Screenshot 2022-08-27 at 00 26 05" src="https://user-images.githubusercontent.com/3165635/186998524-71194f15-1cb3-46bb-9ca4-00517150eb7e.png">

https://deploy-preview-33896--material-ui.netlify.app/base/react-input/

**With the diff**

<img width="1385" alt="Screenshot 2022-08-27 at 00 24 57" src="https://user-images.githubusercontent.com/3165635/186998509-baf98c0e-0b88-4838-9cb8-b6478a22982f.png">